pkg/api/handlers/compat/events.go

```diff
@@ -89,6 +89,10 @@ func GetEvents(w http.ResponseWriter, r *http.Request) {
 			}
 
 			e := entities.ConvertToEntitiesEvent(*evt)
+			if !utils.IsLibpodRequest(r) && e.Type == "image" && e.Status == "remove" {
```

Comment:
y'know, as long as we're re-pushing, why not add a comment here for future maintainers, explaining exactly what the problem is?